cmd/ltagent/init.go

```diff
@@ -103,6 +141,10 @@ func RunInitCmdF(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if err = createTeamAdmins(admin, 10, config); err != nil {
```

Comment:
I'd like this 10 value to be configurable, same as it happens with `NumTeams`. Since we are here, let's do the same for the channels above.